### PR TITLE
Update EIP-4762: Cleanup/refactor the text as per clarifications

### PR DESCRIPTION
**Pull request by g11tech**
cleanup as per the discussions

### CODE REVIEW

File: EIPS/eip-4762.md [hunk 1]
```diff
@@ -221,15 +221,13 @@ When executing a transaction, maintain four sets:
 
 When an **access** event of `(address, sub_key, leaf_key)` occurs, perform the following checks:
 
- * Perform the following steps unless `address` is either:
-   * the transaction's originator;
-   * the transactions's target;
+ * Perform the following steps unless event is a *Transaction access event*;
```
Comment (g11tech) [inline EIPS/eip-4762.md:224]: adress could be txsender or target and event could be some state access, so bettter to clarify the this gas exception is limited to tx events

File: EIPS/eip-4762.md [hunk 2]
```diff
@@ -243,23 +241,6 @@ Note that values should only be added to the witness if there is sufficient gas
  * this minimum 1/64th gas reservation is checked **AFTER** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
  * this 1/64th of the gas is subtracted **BEFORE** charging the witness costs when performing a `CREATE` or `CREATE2`
 
-### Replacement for access lists
```
Comment (g11tech) [inline EIPS/eip-4762.md:246]: accesslists don't need to change structure with the EIP as this info is already available in execution witnesses

**Review by gballet — APPROVED** [review]
LGTM

**Review by Scamreno — COMMENTED** [review]
#8700

**Pull request merged**